### ISSUE TITLE
Ubuntu on Apple Silicon freeze

### ISSUE THREAD

**Issue by hannesa2** [utmapp/UTM#3316]
**Describe the issue**
After reboot of Ubuntu 20.04 arm64 guest it hangs here 
<img width="791" alt="image" src="https://user-images.githubusercontent.com/3314607/141990359-cb5c7ec6-c326-40dd-9f96-6d698b01c37d.png">

**Configuration**
* UTM Version: 2.4.1
* OS Version: macOS 12.0.1
* Intel or Apple Silicon? : Apple Silicon

**Crash log**
There is no crash, only freezing

**Debug log**

[debug.log](https://github.com/utmapp/UTM/files/7546479/debug.log)

**Upload VM**
[config.plist.zip](https://github.com/utmapp/UTM/files/7546513/config.plist.zip)

**Comment by mixman68** [comment]
Same here

**Comment by AwlsomeAlex** [comment]
Upgrade your kernel. This is a known issue with some Ubuntu 20.04 kernels, or use Ubuntu 21.10.

**Comment by hannesa2** [comment]
Hmm, I did always a `sudo apt update && sudo apt dist-upgrade` 
That's why I assume this is not the reason. Sadly I can't verify, because it's no more starting

**Comment by AwlsomeAlex** [comment]
Give it _unfortunately_ about 10 minutes then see if it boots. Otherwise I'd use Ubuntu 21.10, I'm having no issues with it.

**Comment by hannesa2** [comment]
Ok, I gave it 20 minutes, but it still freeze.
What makes me helpless is the fact, that I don't see the reason

**Comment by mixman68** [comment]
I downgrade the graphics to ramfb and the vm boot
I installed desktop and I put back ramfb-gl

Do you have tried this ?

**Comment by hannesa2** [comment]
Ok, changed to this 

<img width="735" alt="image" src="https://user-images.githubusercontent.com/3314607/143059245-26065bba-b189-4e35-9d7c-1a4b78410753.png">

still the same 😞 it looks like this instance is dead for ever.
Sadly it's not the only dead one. What's about to gave us more investigation abilities ?

**Comment by mixman68** [comment]
And I specify in cpu 4 cores 

Envoyé de mon iPhone

**Comment by hannesa2** [comment]
it changed nothing

<img width="777" alt="image" src="https://user-images.githubusercontent.com/3314607/143068474-a9857726-4433-403d-80e1-619d6f0faacb.png">

**Comment by conath** [comment]
@hannesa2 You could try with "Force slower emulation […]" checked (in UTM Pureferences).

**Comment by hannesa2** [comment]
<img width="478" alt="image" src="https://user-images.githubusercontent.com/3314607/143306017-cc3c54ea-d32a-4638-b640-72d43f522950.png">

Sorry it doesn't help

**Comment by conath** [comment]
So it's not a hypervisor or M1 problem and it also doesn't seem to relate to display settings. Maybe an EFI incompatibility? You can try to delete the efi vars drive from the VM; this might cause it to go to EFI Shell but you should be able to Boot From File from the EFI setup menu (type `exit` at the Shell).

I notice you have the CPU set to cortex, I recommend you choose "default" instead. This is recommended to avoid problems on M1.

**Comment by hannesa2** [comment]
I set CPU to `default` (It doesn't help) but the EFI stuff I don't understand. Please explain, maybe with a picture

**Comment by conath** [comment]
Read this entirely before you do it, please.

Here's how to delete the EFI Vars drive:

![delete-vars](https://user-images.githubusercontent.com/12073163/143309331-bbf9c4ee-76ab-4072-a221-922b1da49d65.jpg)

EFI is the most basic level of software in the (virtual) computer. The EFI vars file contains low-level settings like VM screen resolution and which OS if should boot, and a number of other settings. Those might be what's preventing your VM from starting - just a shot in the dark, though. 

Anyway before you go deleting your EFI vars drive, please back up the file. Either make a clone of the VM, or right click the VM in the UTM list of VMs, select Show in Finder, right click the `.utm` file in Finder, Show Package Contents, open Images folder, and copy the `efi_vars.fd` file to somewhere else.

**Comment by hannesa2** [comment]
* I made a clone
* deleted `efi_vars.fs`
* saved
* started VM

But still no success

**Comment by conath** [comment]
Alright then, I am out of ideas. From reading the history here it seems like you had been using the VM for a while. Can you please add the steps to get to this state for your VM? E. g. did this happen after an you installed an update? Which install options did you choose? You mentioned that multiple of your VMs have this problem. Do all of the VMs fail in the same way and after the same actions?

**Comment by hannesa2** [comment]
I installed this VM and it was running for days without any issue. 
After a break of days I tried to start again, without any update, and now it freeze.

Other VM comes with different issue https://github.com/utmapp/UTM/issues/3153

I guess this VM is dead for me. And I see the only solution is to try things out, but I learned that you only can solve problems when you understand it. This is not he case, because there is no good way to investigate, no verbose, nothing. 

I guess you urgent need something like this to have a stable software
